game/gameboard.py

```diff
@@ -71,6 +71,11 @@ def undo_move(self):
         move = self._moves.pop(-1)
         self._matrix[move[0]][move[1]] = None
 
+    def is_valid_move(self, column_index):
+        if self._matrix[0][column_index - 1]:
```

Comment:
@apojomovsky we have been here before: `return self._matrix[0][column_index - 1] is None`